py/etrade-plan-pdf-tx-extract

```diff
@@ -259,7 +259,7 @@ def text_to_rsu_entry(text: str, filename: str) -> BenefitEntry:
          # The sell-to-cover date is almost always a day or two after the release
          # date. This needs to be looked up separately if we want an accurate
          # USD/CAD exchange rate.
-         sell_to_cover_tx_date=None,
+         sell_to_cover_tx_date=data['release date'],
```

Comment:
I don't really get how this is helpful, as your description implies. I didn't see any of use of sell_to_cover_tx_date in logic trying to determine the StCs

The comment above this also now directly contradicts what's happening here.

Reply:
Same here, not needed.

Reply:
See https://github.com/tsiemens/acb/pull/27/files/8c9130545c7b3844fe4f5d224e455aa8464a93e3..0a57482f70583e37e5699598bbec3e6fc917790b